packages/app-content-pages/src/shared/components/Sidebar/Sidebar.js

```diff
@@ -17,16 +17,18 @@ const StyledButton = styled(Button)`
   text-decoration: none;
   color: black;
   padding: 5px 20px; // Same as Project About page sidebar
-  margin-bottom: 5px;
+  width: 100%;
+
   ${props =>
     props.active &&
     css`
-      background: #addde0; // accent-1
-      font-weight: bold;
+      background: ${props.theme.global.colors['accent-1']};
     `}
 
-  &:hover {
-    font-weight: bold;
+  &:hover, :focus {
+    > span {
+      font-weight: bold;
```

Comment:
This is causing the component width to shift/expand when the largest item is hovered (i.e. SOCIAL SCIENCES or ADLER PLANETARIUM) in a noticeable way. Not blocking, and I'm not sure what the solution is 🤔 . Most noticeable in the middle width (~ 1200px).

Reply:
I'm going to leave as is for now. I tried out a couple of quick ideas, but can't find a solution at the moment.